starter_code/views/index.hbs

```diff
@@ -1,3 +1,9 @@
 <div class="index-container">
   
+    <div class="beer-homepage"></div>
+    <div class="btn-home">
+        <a class="button" href="/beers">Check the Beers!</a>
```

Comment:
another way to target the a tags would be to use the selector in your css `.btn-home a {...}` rather than adding the class name button